src/components/common/OfficeAddresses.vue

```diff
@@ -290,7 +290,7 @@ export default class OfficeAddresses extends Mixins(ObjectMixin) {
   private inheritRecDeliveryAddress: boolean = true
 
   // State of the checkbox for determining whether the Record address is the same as the Registered address
-  private inheritRegisteredAddress: boolean = this.isSame(this.registeredAddress, this.recordsAddress)
```

Comment:
This was another instance of "property used before it is initialized".
So I now initialize `inheritRegisteredAddress` in created() below.